nix-in-docker/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM nixos/nix
```

Comment:
I think we should use `nixpkgs/nix-unstable` instead (which is now being updated again). That's a more modern Docker base image

Reply:
Works too 👍